plinth/modules/help/templates/statuslog.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'help_base.html' %}
+{% comment %}
+    #
+    # This file is part of Plinth.
+    #
+    # This program is free software: you can redistribute it and/or modify
+    # it under the terms of the GNU Affero General Public License as
+    # published by the Free Software Foundation, either version 3 of the
+    # License, or (at your option) any later version.
+    #
+    # This program is distributed in the hope that it will be useful,
+    # but WITHOUT ANY WARRANTY; without even the implied warranty of
+    # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    # GNU Affero General Public License for more details.
+    #
+    # You should have received a copy of the GNU Affero General Public License
+    # along with this program.  If not,see
+    <http://www.gnu.org/licenses/>.
+    #
+{% endcomment %}
+
+{% load i18n %}
+
+{% block content %}
+
+<p>
+    {% blocktrans trimmed %}
+        This is the last {{ X }} lines of status log Please attatch with the bug when raising a bug.
```

Comment:
I'd rephrase this a bit ('report a bug', missing point between sentences);
I cannot run the code of this pull request now, so I only look at the text and syntax.